src/app/workspace/resource/resource.component.html

```diff
@@ -69,6 +69,11 @@
 
             </mat-tab>
         </mat-tab-group>
+        <ng-template #deletedResource>
+            <app-properties *ngIf="resource.res" [resource]="resource" [displayProjectInfo]="true"
+                [adminPermissions]="adminPermissions" [editPermissions]="editPermissions" [valueUuidToHighlight]="valueUuid">
```

Comment:
are the `adminPermissions`, `editPermissions`, and `valueUuidToHighlight ` needed here?

Reply:
yes,  because it should be possible (at least in the future) to re-activate (or to erase) a resource. In this case we have to know the permissions in the propertiesComponent

Reply:
I was also wondering if further editing of a resource would eventually be possible. Good to know :)